### ISSUE TITLE
Is the plugin deprecated?

### ISSUE THREAD

**Issue by nicoxb** [pseudomuto/protoc-gen-doc#543]
Has this plugin stopped being maintained? It has not been released for more then 2 years, And some PRs were not processed.

**Comment by fmontorsi-equinix** [comment]
yeah I wonder the same thing... I would like to get something fixed and I can open a PR but I'm not sure if that will be processed...

@pseudomuto  do you still have bandwidth to maintain this (very useful!) tool? thanks!

**Comment by pseudomuto** [comment]
My apologies for the late reply. It is not abandoned! I haven't spent much time on it lately to be honest as it largely seemed to server its purpose. Could definitely use some TLC though and I'd happily review PRs for fixes.
